src/applications/ivc-champva/shared/components/fileUploads/supportingDocsVerification.js

```diff
@@ -35,7 +35,7 @@ export default class SupportingDocsVerification {
           const itemProps = Array.isArray(items)
             ? items[0]?.properties // Confirmation page has different page structure
             : items?.properties;
-          return Object.keys(itemProps).map(
+          return Object.keys(itemProps ?? {}).map(
```

Comment:
Fixes edge case where this `map` would break if the user somehow landed on the confirmation page first and clicked submit. Should only be possible in development, but wanted to patch anyway.